k8s/k8s.go

```diff
@@ -250,6 +263,118 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, hostname string, calicoCl
 	return result, nil
 }
 
+// ipAddrsResult parses the ipAddrs annotation and calls IPAM plugin for
+// each IP passed to it by setting CNI_ARGS IP=ip_addr, and returns the type.Result.
+// Example annotation value string: "[\"10.0.0.1\", \"2001:db8::1\"]"
+func ipAddrsResult(ipAddrs string, conf utils.NetConf, args *skel.CmdArgs, logger *log.Entry) (*types.Result, error) {
+	var ips []string
+
+	err := json.Unmarshal([]byte(ipAddrs), &ips)
```

Comment:
This code to parse a string into a list of IP addresses is also in the overrides function below.

It would be nice if both `ipAddrsResult` and `overrideIPAMResult` took `[]net.IP` as an arg rather than a string.  We could make the parsing code common in a utility function.